docs/installation/octopus-in-container/octopus-server-container-linux.md

```diff
@@ -1,23 +1,21 @@
 ---
-title: Octopus Deploy Server Container Linux
+title: Octopus Deploy Server Linux Container
 description: An Octopus Deploy Server instance can be run directly from within a container.
 position: 1
 ---
 
 If you want to run an Octopus Deploy Windows container, please refer to the [Octopus Deploy Server Container Windows](/docs/installation/octopus-in-container/octopus-server-container-windows.md) documentation.
 
+**Octopus Deploy Linux Containers are part of our Early Access Program (EAP) and may contain bugs or be unstable.**
+
+Note: When using Linux containers on a Windows machine, please ensure you have [switched to Linux Containers](https://docs.docker.com/docker-for-windows/#switch-between-windows-and-linux-containers).
```

Comment:
I think this might go better in a troubleshooting section, especially if we include a googleable error message